web/v2/backend/src/main/java/com/tourneygen/web/Models/Team.java

```diff
@@ -173,4 +174,16 @@ public void merge(
                     () -> new EntityNotFoundException("League with id " + id + " was not found"));
     this.roster = teamUpdateDTO.getRoster() == null ? this.roster : teamUpdateDTO.getRoster();
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
overriding `equals` requires overriding`hashcode`.

I think I saw earlier that we're using maven. Look into the Commons Equals and Hashcode builders for doing this.

Reply:
Hashcode is done by the @Entity with the JPA functionality. When i change it it breaks the JPA functionality.

Reply:
Fair enough